common/versioning/version.properties

```diff
@@ -1,3 +1,3 @@
 #Thu Aug 02 12:03:16 PDT 2018
-versionName=0.0.12
+versionName=0.0.14-RC1
```

Comment:
13?

Reply:
We already used 13 for teams/msal release from `dev` branch. I will use 14 for this and 1.0.0 for msal GA release when cut a release branch